README.md

```diff
@@ -3,6 +3,34 @@
 [Homebridge Net Nanny](https://github.com/pricemi115/homebridge-grumptech-netnanny), by [GrumpTech](https://github.com/pricemi115/), is a [Homebridge](https://homebridge.io) dynamic platform plug-in that publishes metrics measuring the health of a network.
 This plugin was inspired by [Dave Hamilton's](https://twitter.com/DaveHamilton), of the [Mac Geek Gab](https://www.macgeekgab.com), 3-Ping strategy for evaluating network health.
 
+## What Problem Does Net Nanny Solve?
+
+When we suffer network problems, it's sometimes hard to figure out whether the problem is on the local area network, a problem with your Internet service provider, or a problem with a service to which you're trying to connect. In order to isolate the problem, a good practice is to open a terminal window and ping your router, then your modem, and then a known-reliable server on the Internet.
+
+The Net Nanny Homebridge plugin is designed to give you this information at a glance from within HomeKit. The Homebridge plugin creates a fake switch called Net Nanny, along with fake carbon dioxide (C02) sensors that will report your network health.
```

Comment:
The services created by the plugin accessory:
- One switch named "Export History" but the Home app shows the name as "NetNanny Switches". The name is shown correctly in Homebridge.  
- For each target
--> One switch named "Power <Target name>" but the Home app shows the name as the Target name.
--> (3) CO2 sensors that report network health.